README.md

```diff
@@ -29,28 +29,41 @@ cf restage $YOUR_APP_NAME
 
 #### Log Collection (Beta - No official release yet)
 
-To start collecting logs from your application in CloudFoundry, the agent contained in the buildpack needs to be activated and log collection enabled.
+To start collecting logs from your application in CloudFoundry, use the following configuration:
 
 ```
+# activate the agent
 cf set-env $YOUR_APP_NAME RUN_AGENT true
+# enable log collection
 cf set-env $YOUR_APP_NAME DD_LOGS_ENABLED true
-# Disable the Agent core checks to disable system metrics collection
+# configure the agent to collect logs from stderr and stderr
```

Comment:
It's unclear to me than any of those two lines would configure it, I'd make it clearer